### PR TITLE
Tableau permissions: add "View All" to menu

### PR DESCRIPTION
**Pull request by orangejenny**
## Product Description
Followup for https://github.com/dimagi/commcare-hq/pull/31659

https://dimagi-dev.atlassian.net/browse/QA-4193

## Feature Flag
Embedded Tableau

### Safety story
This was originally a qabug and will be tested before release.

### Automated test coverage

None

### QA Plan

QA team will verify this ticket

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/tabs/tabclasses.py [hunk 1]
```diff
@@ -277,25 +277,30 @@ def _get_saved_reports_dropdown(self):
...
-                items.extend([
-                    dropdown_dict(report.name, url=reverse(TableauView.urlname, args=[self.domain, report.id]))
-                    for report in reports
-                ])
+                tableau_reports = [(_('Tableau Reports'), [
+                    {
+                        'show_in_dropdown': i < 2,
+                        'title': report.name,
+                        'url': reverse(TableauView.urlname, args=[self.domain, report.id]),
+                    } for i, report in enumerate(reports)
```
Comment (AddisonDunn) [inline corehq/tabs/tabclasses.py:295]: 😍

File: corehq/tabs/tabclasses.py [hunk 2]
```diff
@@ -277,25 +277,30 @@ def _get_saved_reports_dropdown(self):
...
+                ])]
```
Comment (AddisonDunn) [inline corehq/tabs/tabclasses.py:296]: 😍

**Review by AddisonDunn — APPROVED** [review]

**Review by snopoke — APPROVED** [review]

File: corehq/tabs/tabclasses.py [hunk 3]
```diff
@@ -277,25 +277,30 @@ def _get_saved_reports_dropdown(self):
...
+            reports = TableauVisualization.for_user(self.domain, self.couch_user)
             if reports:
-                items.append(dropdown_dict(_('Tableau Reports'), is_header=True))
-                items.extend([
-                    dropdown_dict(report.name, url=reverse(TableauView.urlname, args=[self.domain, report.id]))
-                    for report in reports
-                ])
+                tableau_reports = [(_('Tableau Reports'), [
+                    {
+                        'show_in_dropdown': i < 2,
```
[the remark this answers is not in the document]
Comment (esoergel) [reply corehq/tabs/tabclasses.py]: Ah neat, thanks for the explanation

**Review by esoergel — APPROVED** [review]
looks great!

**Comment by orangejenny** [comment]
Merging this so it makes the deploy. There's one lingering question on the ticket, but I'm reasonably confident it isn't related to this change.

**Pull request merged**